docs/examples/datepicker/ExEvents.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+    <span>
+        <b-field>
+            <b-switch v-model="bars">Bars</b-switch>
+        </b-field>
+        <b-datepicker
+            inline
+            v-model="date" 
+            :events="events"
+            :indicators="indicators"
+            >
+        </b-datepicker>
+    </span>
+</template>
+
+<script>
+    export default {
+        computed: {
+            indicators() {
+                return this.bars ? 'bars' : 'dots'
+            }
+        },
+        data() {
+            return {
+                date: new Date(2017, 10, 1),
```

Comment:
It would be nice to have events in the current month and no a fixed month (10 -> november).